### DISCUSSION TITLE
Dynamic Score Multiplier for Difficulty Adjust

### DISCUSSION THREAD

**Discussion by WalmartWhiteCat** [ppy/osu#25630] [Ideas] [answered]
Okay, this is like, a personal thing to start off the discussion, but Difficulty Adjust halving the score regardless of the actual values you set with it just annoys me to no end. So I propose a simple solution, so simple that it might have already been considered by the dev team and others outside of the team. And I promise I already searched discussions and found nothing that talked about this, so I sincerely apologise if this is yet another dupe (If you couldn't tell, I'm bad at using Github).

Okay, so basically, just make it so if none of the values are changed then neither is the score multiplier. When values are changed, depending on how much easier or harder the map becomes, the score multi changes (Easier lowers the multi, harder increases it). Incase it isn't clear, basically make it like DT's Speed Increase slider, the faster the speed the higher the multi, the slower the speed the lower the multi.

**Comment by bdach** [comment, marked as answer]
> And I promise I already searched discussions and found nothing that talked about this

https://github.com/ppy/osu/discussions/17461
https://github.com/ppy/osu/discussions/24961
https://github.com/ppy/osu/discussions/24608

among others.

> When values are changed, depending on how much easier or harder the map becomes, the score multi changes

The primary issue here is AR. Lower AR does not necessarily make maps easier.

**Reply by WalmartWhiteCat** [reply]
Would it be possible to make certain changes to diff stats less weighted for score multi? Like, for instance, AR, like you said. Everyone knows AR is kind of like a preference, until you get to obscenely low or obscenely high values anyway. What about, for stats that don't necessarily make things easier or harder, but instead are mostly preference, the multiplier change is almost nothing until 7 (arbitrary)? After it hits a point where it stops being preference and starts actually affecting difficulty to an objective degree, the score multi changes more and more. For example, 7-10AR brings no change, then for every 1AR (arbitrary) value that is dropped below 7 the multi rises a little. Also do the same for higher than 10 but in smaller increments, like every .2 (arbitrary) the intensity gets stronger but to a lesser degree than lowering the AR.
